.travis.yml

```diff
@@ -2,7 +2,7 @@ sudo: required
 dist: trusty
 language: node_js
 node_js:
-  - stable
+  - 10.5.0
```

Comment:
The latest stable version(12.10.0) always make the CI build failed when run with `eval yarn`.